File: OfficeFinder/app/src/main/java/sk/stuba/fiit/officefinder/models/Parking.kt
package sk.stuba.fiit.officefinder.models

import com.google.gson.annotations.SerializedName

class Parking (
        @SerializedName("name")
        val name: String,
        @SerializedName("id")
        val id: Int,
        @SerializedName("geoPointStrings")
        val geoPointStrings: List<GeoPointString>,
        @SerializedName("distance")
        val distance: Double,
        @SerializedName("score")
        val score: Double,
        @SerializedName("access")
        val access: String
)

File: OfficeFinderBackend/OfficeFinderBackend/Models/Parking.cs
using System.Collections.Generic;

namespace OfficeFinderBackend.Models
{
   public class Parking
   {
      public long Id { get; set; }
      public string Name { get; set; }
      public string GeoJson { get; set; }
      public IEnumerable<GeoPointString> GeoPointStrings { get; set; }
      public string Access { get; set; }
      public double Distance { get; set; }
      public double Score { get; set; }
   }
}